doc/lua_api.txt

```diff
@@ -5400,7 +5399,14 @@ This is basically a reference to a C++ `ServerActiveObject`
 
 * `get_pos()`: returns `{x=num, y=num, z=num}`
 * `set_pos(pos)`: `pos`=`{x=num, y=num, z=num}`
-* `move_to(pos, continuous=false)`: interpolated move
+* `move_to(pos, continuous=false)`
+    * Does an interpolated move for lua entities. The object will not instantly
```

Comment:
Suggestion for both sentences:
`Does an interpolated move for Lua entities for visually smooth transitions.`

Reply:
Thanks! This is much better.